storey/redis_driver.py

```diff
@@ -70,10 +70,13 @@ def __init__(self,
         # upon demand (any access to self.redis)
         if redis_client is not None:
             self._redis = redis_client
+            if isinstance(redis_client, redis.cluster.RedisCluster):
+                redis_type = RedisType.CLUSTER
+            if isinstance(redis_client, redis.Redis):
+                redis_type = RedisType.STANDALONE
         else:
             NeedsRedisAccess.__init__(self, redis_url)
             self._redis = None
-
```

Comment:
I think this newline is actually appropriate here.

Reply:
ok